src/main/java/com/autotune/utils/TrialHelpers.java

```diff
@@ -420,21 +239,22 @@ public static ExperimentTrial createDefaultExperimentTrial(int trialNumber,
             }
         }
 
-        for (String tracker : trackers ) {
+        for (String tracker : trackers) {
             TrialDetails deployment = new TrialDetails(tracker,
                     autotuneExperiment.getDeploymentName(),
                     autotuneExperiment.getApplicationDeployment().getNamespace(),
-                    "",
-                    TRIAL_RUNNING,
-                    TRIAL_RUNNING,
-                    TRIAL_RUNNING,
```

Comment:
Whys is this being removed from the constructor? This is being used by the API to indicate that the trial is still ongoing

Reply:
Eventually those parameters will coming up , Thinking for decoupling TrialDetails object from current status

Reply:
done